src/app/core/cases/cases.service.ts

```diff
@@ -168,6 +169,26 @@ export class CasesService {
       });
   }
 
+  getCaseHistoryView(jurisdictionId: string,
+              caseTypeId: string,
+              caseId: string,
+              eventId: string): Observable<CaseHistoryView> {
+    const url = this.appConfig.getApiUrl()
```

Comment:
can we move the construction of the url inside a appConfig.getCaseHistoryUrl()?

Reply:
Done